### ISSUE TITLE
tsh on windows reports errors about missing webauthn.dll 

### ISSUE THREAD

**Issue by rosstimothy** [gravitational/teleport#22833]
The warnings are shown even when mfa isn't being used:

```bash
tsh.exe version
time="2023-03-09T13:42:24Z" level=warning msg="WebAuthnWin: failed to check version" error="A device attached to the system is not functioning."
Teleport v12.1.0 git:api/v12.1.0-0-g49a6de744a go1.19.6
```

When establishing a session via `tsh ssh` or Connect the warning is also present even though mfa is successful.

**Comment by webvictim** [comment]
I'm having a similar issue on Windows Server 2019:

Similar to #22833 but not exactly the same.

```
PS C:\Users\alice.teleporter> .\tsh.exe version
time="2023-03-16T19:50:48Z" level=warning msg="WebAuthnWin: failed to check version" error="Failed to find WebAuthNGetApiVersionNumber procedure in WebAuthn.dll: The specified procedure could not be found."
Teleport v11.3.8 git:api/v11.3.8-0-g516d592567 go1.20.2
Proxy version: 11.3.7
Proxy: gus.teleportdemo.com:443
```

The error appears in the console basically any time you run a `tsh` command.

![image](https://user-images.githubusercontent.com/897821/225737416-49130572-41dc-415d-9e73-63f9fb1c9161.png)

I'll report back after 11.3.9 is released to see whether it fixes it.

**Comment by ravicious** [comment]
"Failed to find WebAuthNGetApiVersionNumber procedure in WebAuthn.dll" described by @webvictim has been fixed in #38444.

**Comment by sergefedorow** [comment]
I have the same issue with tsh v.14.3.3 and 15.0.2.
Client is Windows 10 Enterprise Evaluation, version 1809
Installation did from zip file. 
```
C:\Users\IEUser>tsh version
panic: Failed to find WebAuthNGetApiVersionNumber procedure in WebAuthn.dll: The specified procedure could not be found.

goroutine 1 [running]:
golang.org/x/sys/windows.(*LazyProc).mustFind(...)
        C:/Users/runneradmin/go/pkg/mod/golang.org/x/sys@v0.16.0/windows/dll_windows.go:325
golang.org/x/sys/windows.(*LazyProc).Addr(...)
        C:/Users/runneradmin/go/pkg/mod/golang.org/x/sys@v0.16.0/windows/dll_windows.go:333
github.com/gravitational/teleport/lib/auth/webauthnwin.webAuthNGetApiVersionNumber()
        C:/a/teleport.e/teleport.e/lib/auth/webauthnwin/zsyscall_windows.go:83 +0xb4
github.com/gravitational/teleport/lib/auth/webauthnwin.newNativeImpl()
        C:/a/teleport.e/teleport.e/lib/auth/webauthnwin/webauthn_windows.go:65 +0xad
github.com/gravitational/teleport/lib/auth/webauthnwin.init()
        C:/a/teleport.e/teleport.e/lib/auth/webauthnwin/webauthn_windows.go:36 +0x169
```
